csharp/AlgorithmWorkshop/QuickSort.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace AlgorithmWorkshop
+{
+    public static class QuickSort
+    {
+        public static List<double> Sort(List<double> listToSort)
```

Comment:
It would be great to have this work for more than `double`, perhaps using [generics](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/generics/).